lib/utils/documentdb.ts

```diff
@@ -540,6 +579,30 @@ export function replaceDocument<T>(
   });
 }
 
+/**
+ * Deletes a document
+ *
+ * @param documentLink  the _self link to the document
+ */
+export function deleteDocument(
```

Comment:
what we need this for?

Reply:
that's for [this one](https://github.com/teamdigitale/digital-citizenship-functions/pull/312/files#diff-1a49c09fccc54d018f5de1b082ecc1acR80)

I'd pause this PR until we converge on a safe approach for #312